packages/mdc-select/component.ts

```diff
@@ -37,9 +37,9 @@ import {MDCSelectHelperText, MDCSelectHelperTextFactory} from './helper-text/ind
 import {MDCSelectIcon, MDCSelectIconFactory} from './icon/index';
 import {MDCSelectEventDetail, MDCSelectFoundationMap} from './types';
 
-type PointerEventType = 'mousedown' | 'touchstart';
+type PointerEventType = 'click';
```

Comment:
We not longer need `POINTER_EVENTS` & `PointerEventType` since this is only single event, we can directly add register it to handleClick foundation method without looping through `POINTER_EVENTS`.